crates/re_log_encoding/src/decoder.rs

```diff
@@ -29,6 +31,12 @@ pub enum DecodeError {
     #[error("Not an .rrd file")]
     NotAnRrd,
 
+    #[error("Old .rrd file")]
+    OldRrdVersion,
```

Comment:
More information would be nice - what version did we find, what version is supported
Pointing this out because we apparently have bad errors on old rrd versions given the kind of questions etc. I see on Discord